src/Query/User/PackageQuery/DbalPackageQuery.php

```diff
@@ -143,7 +143,8 @@ public function getById(string $id): Option
                 last_scan_date,
                 last_scan_status,
                 last_scan_result,
-                keep_last_releases
+                keep_last_releases,
+                readme
```

Comment:
:thinking: since it will be displayed in one place anyway, we could do a separate read model for it ...
one could even slim down the current model `Package` and create `PackageDetails` with readme, install and other this that are required **only** on package details page.

Reply:
getByID() is only used in places where the package ID is a route parameter. These is all the frontend package 'action' pages, and the webhook endpoint.

Out of those routes, the package details is going to be the most often used. If PackageDetails was split out, then this page would be querying the same table twice.

Unless I create a new route model binding (I come from Laravel, so not sure of the Symfony naming) for PackageDetails, and have that model extend Package.

Reply:
You can create `PackageDetailsParamConverter` for that (look at `PackageParamConverter`)